docs/reference/env_variables.rst

```diff
@@ -0,0 +1,177 @@
+.. meta::
+    :description: HIP environment variables reference
+    :keywords: AMD, HIP, environment variables, environment, reference
+
+*************************************************************
+HIP environment variables
+*************************************************************
+
+In this section, the reader can find all the important HIP environment variables. The full collection of the ROCm environment variables is on the :doc:`ROCm environment variables page<rocm:reference/env-variables>`.
+
+GPU isolation variables
+=======================
+
+The GPU isolation environment variables in HIP are collected in the next table. For more information, check :doc:`GPU isolation page <rocm:conceptual/gpu-isolation>`.
+
+.. list-table::
+    :header-rows: 1
+    :widths: 70,30
+
+    * - **Environment variable**
+      - **Usage**
+
+    * - | ``ROCR_VISIBLE_DEVICES``
+        | A list of device indices or UUIDs that will be exposed to applications.
+      - Example: ``0,GPU-DEADBEEFDEADBEEF``
```

Comment:
Is GPU-DEADBEEFDEADBEEF and actual UUID or is it intended to represent something else?

Reply:
I think it's funny and kudos for the original author. I can change it to a more realistic one.